Classes/Controller/MailqueueModuleController.php

```diff
@@ -67,6 +69,7 @@ public function __invoke(Message\ServerRequestInterface $request): Message\Respo
 
         // Force redirect when page selector was used
         if ($request->getMethod() === 'POST' && !isset($request->getQueryParams()['page'])) {
+            $this->assertAllowedHttpMethod($request, 'POST');
```

Comment:
The security check is superfluous here; we already verify and allow POST in the line above.